examples/amp-story/quiz.html

```diff
@@ -69,6 +69,47 @@ <h2>True or False: Cats can be right-pawed or left-pawed.</h2>
           </amp-story-quiz>
         </amp-story-grid-layer>
       </amp-story-page>
+      <amp-story-page id="quiz-question-3">
+        <amp-story-grid-layer template="fill">
+            <amp-img src="https://i.pinimg.com/originals/8d/b1/e9/8db1e9b7cc3ee52b13cabe3a53c62f05.jpg"
+            width="720" height="1280"
+            layout="responsive">
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="vertical">
+          <style>
+            #cat-question-cute {
+              position: absolute;
```

Comment:
Could you document all the ways to position a quiz that a publisher would have, and which one is the preferred one?
The issue with absolute positions is that they're not really responsive, even if using `%` top/right/bottom/left properties.

Reply:
I'd like to discuss some of the options we have here - specifically using `grid-area` to place the quiz on the top / middle / bottom third of the story-page - right now adding the `grid-area` attribute seems to remove the quiz entirely.

Reply:
So I talked with Jon a bit about this, I think the best way to go is to centering the quiz automatically both vertically using `grid-area="middle-third"` and horizontally, and allowing the publisher to position the quiz in less conventional ways using `position: absolute` (presumably with percentages instead of pixel values).

I'll add documentation to make this apparent

Reply:
Sorry, I might have been a little unclear; you can use `middle-third` in general, but it won't necessarily vertically center the element.  It will start the element in the middle third of the screen.  But e.g. if the element is 100% of the height of the page, it won't vertically center it; it will push it 33% down the page and the bottom 33% of the element will be cut off.

I agree vertically centering it by default could look nice, but using thirds is probably not the way to do it.  If this is meant to eventually have its own type of layer as specified in the design, can we just center it using flexbox on the layer?

Reply:
I just came to the same conclusion looking into this, instead I'm opting for vertically centering on the layer with flexbox.